python/TestHarness/RunParallel.py

```diff
@@ -220,7 +220,8 @@ def spinwait(self, time_to_wait=0.05):
             threshold = max(start_min_threshold, (0.1 * float(tester.specs['max_time'])))
 
             if now >= threshold:
-              self.harness.handleTestResult(tester, '', 'RUNNING...', start_time, now, False)
+              tester.setStatus('RUNNING...', 'PENDING')
+              self.harness.handleTestResult(tester, '', '', start_time, now, False)
```

Comment:
TODO: remove the result variable from handleTestResult(tester, output, result, ,). I think we no longer need to pass that around.